src/back/msl/mod.rs

```diff
@@ -82,7 +82,7 @@ pub type EntryPointResourceMap = std::collections::BTreeMap<String, EntryPointRe
 enum ResolvedBinding {
     BuiltIn(crate::BuiltIn),
     Attribute(u32),
-    Color(u32),
+    Color(u32, Option<u32>),
```

Comment:
nitpick: I don't think it's obvious what these fields mean any more; named fields are probably a better fit here.